app/services/create_batch.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBatch
+  def call
+    appointment_summaries = AppointmentSummary
+                            .includes(:appointment_summaries_batches)
+                            .where('appointment_summaries_batches.appointment_summary_id': nil)
+
+    return nil if appointment_summaries.empty?
+
+    Batch.create(appointment_summaries: appointment_summaries)
```

Comment:
If this fails do we expect the caller to know the internals of the `call` method or should the caller provide a failure method to invoke?